### PR TITLE
FOGL-8982 Add documentation on allow and deny lists

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
Checkin only with screenshot?
Add some details for the firewall in secure fledge page.

Also, this screenshot needs to be updated once GUI issue FOGL-8978 is resolved. 
Like Display name and description is Not appearing for list type on configuration page.

**Comment by MarkRiddoch** [comment]
> Checkin only with screenshot? Add some details for the firewall in secure fledge page.
> 
> Also, this screenshot needs to be updated once GUI issue FOGL-8978 is resolved. Like Display name and description is Not appearing for list type on configuration page.

Whoops, sorry, forgot to add the other file to the commit.

File: docs/securing_fledge.rst [hunk 1]
```diff
@@ -90,6 +91,25 @@ After enabling HTTPS and selecting save you must restart Fledge in order for the
...
+| |firewall| |
++------------+
+
+Clicking on the arrow icon beside each list will expand the list and show the current contents of the list.
+
+Click on the *Add new item* link to create a new entry in the list.
+
+To remove an entry from the list click on the *x* icon to the right of the list item.
+
+If the allow list is non-empty, then any access, including ping, to the Fledge API port will be checked to see if the source IP address of the request matches an entry in the allow list. If the address of the requester is not in this allow list then the API will not send any response to the caller andthe connection will be closed. The only address that is exempt from this checking is the localhost via the loopback interface, 127.0.0.1. This is required for local management of the Fledge instance and must always be accessible.
```
Comment (ashish-jabble) [inline docs/securing_fledge.rst:109]: space required - `andthe`

File: docs/securing_fledge.rst [hunk 2]
```diff
@@ -90,6 +91,25 @@ After enabling HTTPS and selecting save you must restart Fledge in order for the
 | |connection_https| |
 +--------------------+
 
+Allow & Block Lists
```
Comment (ashish-jabble) [inline docs/securing_fledge.rst:94]: Block -> Deny (Consistency wise)

File: docs/securing_fledge.rst [hunk 3]
```diff
@@ -90,6 +91,25 @@ After enabling HTTPS and selecting save you must restart Fledge in order for the
...
+
++------------+
+| |firewall| |
++------------+
+
+Clicking on the arrow icon beside each list will expand the list and show the current contents of the list.
+
+Click on the *Add new item* link to create a new entry in the list.
+
+To remove an entry from the list click on the *x* icon to the right of the list item.
```
Comment (ashish-jabble) [inline docs/securing_fledge.rst:107]: It will be better if we add numbered bullet points for each

**Comment by ashish-jabble** [comment]
@MarkRiddoch Would you mind to replace the screenshot with below? This is created from GUI branch FOGL-8978, as display name, description, order handling is missing in develop for the list type config items on Configuration page.

firewall

**Pull request merged**